04-spatial-operations.Rmd

```diff
@@ -559,18 +570,20 @@ The resulting `nz_avheight` objects have the same geometry as the aggregating ob
 Note that regions containing no points have an associated `elevation` value of `NA`.
 For aggregating operations which also create new geometries, see section \@ref(geometry-unions).
 
+<!-- One should avoid to have a single subsection, here 4.2.5.1. I would delete the heading, then spatial congruence just becomes another paragraph of spatial aggregation -->
```

Comment:
One should avoid to have a single subsection, here 4.2.5.1. I would delete the heading, then spatial congruence just becomes another paragraph of spatial aggregation

Reply:
Agreed. Job done.